akka-docs/src/test/scala/docs/stream/operators/sink/Fold.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sink
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.{ Sink, Source }
+
+import scala.concurrent.Future
+
+object Fold {
+  implicit val system: ActorSystem = ???
+
+  def foldExample: Future[Int] = {
+    //#fold
+    Source(1 to 100).runWith(Sink.fold(0)(_ + _))
```

Comment:
I think it'd that although the _ + _ is fine in general it would be good to give the parameters names in an example as they help understand what is what. Names should be the same as in the Java example.